apparmor.txt

```diff
@@ -38,6 +38,7 @@ profile hassio-supervisor flags=(attach_disconnected,mediate_deleted) {
   /data/** rw,
   /run/** rwk,
   /dev/tty rw,
+  /root/.cas/** rwk,
```

Comment:
```suggestion
  /root/.cas/{,**} rwk,
```
Think it has to be this given this is the error:
```
22-02-23 18:32:35 ERROR (MainThread) [supervisor.utils.codenotary] Can't parse CodeNotary output: b'mkdir /root/.cas: permission denied\n' - Expecting value: line 1 column 1 (char 0)
```
You're giving read write permission to everything within `.cas` but it still won't be able to make the actual `/root/.cas` folder.

Reply:
Oh wait I get it. You're adding `/root/.cas/config.json` to the image in https://github.com/home-assistant/supervisor/pull/3462. Therefore the folder will already exist and cas won't try and make it at runtime anymore. Nvm, this is fine then.